src/main/kotlin/se/uulm/snowballr/backend/model/dto/Project.kt

```diff
@@ -57,3 +57,12 @@ fun Project.toGrpcProject(): ProjectOuterClass.Project {
         .setSettings(settings)
         .build()
 }
+
+/**
+ * Creates a list of [ProjectOuterClass.Project]s from a list of [Project]s
+ */
+fun toGrpcProjects(projects: List<Project>): ProjectOuterClass.Project.List {
+    val builder = ProjectOuterClass.Project.List.newBuilder()
+    projects.forEach { builder.addProjects(it.toGrpcProject()) }
+    return builder.build()
+}
```

Comment:
By using an extension function similar to the above, the call reads more naturally.
```suggestion
/**
 * Creates a list of [ProjectOuterClass.Project]s from this list of [Project]s.
 */
fun List<Project>.toGrpcProjects(): ProjectOuterClass.Project.List {
    val builder = ProjectOuterClass.Project.List.newBuilder()
    this.forEach { builder.addProjects(it.toGrpcProject()) }
    return builder.build()
}
```

This can be call like this:
```kotlin
projects.toGrpcProjects()
// instead of
toGrpcProjects(projects)
```

Reply:
I have already tried this myself but failed, now I know how to do it, thank you!